src/Forms/Shared/Samples/GeometryEngine/CreateGeometries/CreateGeometries.xaml

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"             
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms"
+             xmlns:mapping="clr-namespace:Esri.ArcGISRuntime.Mapping;assembly=Esri.ArcGISRuntime" 
+             x:Class="ArcGISRuntime.Samples.CreateGeometries.CreateGeometries">
+  <Grid>
+    <esriUI:MapView x:Name="MyMapView">
+      <mapping:Map/>
```

Comment:
It looks like the map is later re-created in code. I think we can leave this out, and turn the mapview declaration into one line, `<esriUI:MapView x:Name="MyMapView" />`.